kafka-streams-partitioners/weighted-group-partitioner/build.gradle.kts

```diff
@@ -26,4 +25,5 @@ dependencies {
   testImplementation("org.junit-pioneer:junit-pioneer:1.7.1")
   testImplementation("org.mockito:mockito-core:4.5.1")
   testRuntimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.17.2")
+  testRuntimeOnly("io.grpc:grpc-netty:1.45.1")
```

Comment:
you really shouldn't need the netty runtime for your tests - this would be required if using real channels, prefer mocks.